M2/Macaulay2/d/actors3.d

```diff
@@ -730,23 +730,23 @@ sin(e:Expr):Expr := (
      is x:CCcell do toExpr(sin(x.v))				    -- # typical value: sin, CC, CC
      is x:RRcell do toExpr(sin(x.v))				    -- # typical value: sin, RR, RR
      is x:RRicell do toExpr(sin(x.v))				    -- # typical value: sin, RRi, RRi
-     else buildErrorPacket("expected a number")
+     else WrongArgRRorRRiorCC()
```

Comment:
I'm fine with everything in this PR except the uses of `WrongArgRRorRRiorCC`. I don't think that function should exist. Either the error should be less informative (e.g. just say `number` or `real or complex number` or call `WrongArgRRorCC`) or there should be a more flexible alternative, like `WrongArg(RR,RRi,CC)` which can be easily adjusted.

Reply:
Fair enough -- `WrongArgRRorCC` works for me!

This is all pretty minor anyways since the vast majority of the time, users will be using the methods that call these functions and never see these error messages anyway.

Reply:
> This is all pretty minor anyways since the vast majority of the time, users will be using the methods that call these functions and never see these error messages anyway.

Exactly, which is why I prefer minimal and maintainable interpreter code over verbose.